pandas/indexes/multi.py

```diff
@@ -82,8 +84,6 @@ def __new__(cls, levels=None, labels=None, sortorder=None, names=None,
         if len(levels) == 1:
             if names:
                 name = names[0]
-            else:
```

Comment:
why are you removing this? ``names``/``name`` are equivalent (I *think* this is taken care above), pls confirm.